geowebcache/nestedstorage/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
```

Comment:
@jodygarnett Any suggestions on how to split this into two pull requests, I believe cherry picking is problematic and frowned upon?  As I said I am relatively new to git pull requests